.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,14 @@
-*Issue #, if available:*
 
-*Description of changes:*
+#### Background
+* What do these changes do? 
+* Why are they important?
 
+#### Testing
+* How did you test these changes?
 
+#### Links
+* Links to additional context, if necessary
+* Issue #, if applicable
```

Comment:
Nice to have: notes about how to auto-close issues (with [link to GH docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))